### PR TITLE
adding channel get all

### PR DESCRIPTION
**Pull request by pteyssedre**
Hi,

Thanks for the nice work on the octopus-deploy ! I added the channel API call that I need for one of my project So I thought it could be useful to others

### CODE REVIEW

File: test/mocks/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 module.exports = {
   ...require('./deployment-process'),
+  ...require('./channel'),
```
Comment (parkerholladay) [inline test/mocks/index.js:3]: Nit pick: Sort imports...

**Review by parkerholladay — CHANGES_REQUESTED** [review]
This is looking good. It looks like a `.tgz` file was committed that needs to be removed before we can merge. Also, if you wouldn't mind updating the `README` to include your new endpoint, that'd be great.

**Comment by parkerholladay** [comment]
@pteyssedre Make sure that when you remove the `tgz` file, it isn't left in history either. Thanks for the PR.

**Comment by pteyssedre** [comment]
I just have one question about the repo. I'm unable to run the test because of some errors due to  `AssertionError: expected file to have been called with arguments embrace/justice/tyrael.hots, { name: "tyrael.hots" }`

Is there anything I need to do to be able to run them properly ?

**Comment by parkerholladay** [comment]
Sorry for not being very responsive. There isn't anything special needed to run tests other than installing dependencies, and running `npm test`. But, they seem to all be passing, and your new test is running fine.

**Review by parkerholladay — APPROVED** [review]

**Review by parkerholladay — APPROVED** [review]

**Review by parkerholladay — APPROVED** [review]

**Review by parkerholladay — APPROVED** [review]

**Review by parkerholladay — APPROVED** [review]

**Comment by parkerholladay** [comment]
Rebased and merged, but github is experiencing issues. Closing PR.

**Pull request closed without merging**